conf/metafactory.lua

```diff
@@ -56,7 +56,8 @@ function basic(a)
             local set = mcs.set(prefix, num, 0, 30, size)
             mcs.write(set)
             mcs.flush()
-            local res = mcs.read()
+            local set_res = mcs.res_new()
```

Comment:
think you misunderstood my request:

- Move `local set_res = res_new()` to where line 45 is.
- This allows reuse of the object on each execution run.

Reply:
Oh right, because it's in a function that's being returned. 
Sorry about that! Please let me know if I need to make changes elsewhere, thanks!